docs/backup-restore.md

```diff
@@ -0,0 +1,106 @@
+# Scalar DL Backup Creation and Restoration
+
+Since Scalar DL provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
```

Comment:
Please don't just copy... It's an explanation for Scalar DB so replacing Scalar DB with Scalar DL makes the sentence weird for sure.

```suggestion
Since Scalar DL uses Scalar DB that provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
```

Reply:
Sorry for the inconvenience.
Document updated with new structure and sentences.